### PR TITLE
Fixes #12 - Log level needs to be configurable

### LINKED ISSUE #12: Log level needs to be configurable
The log level needs to be configurable through the env_sample.json file. So these may need to be moved around a little. Otherwise it prints up debug messages.

https://github.com/tl-its-umich-edu/unizin-validation/blob/814f1390a5f0f0c315e7f519e920e7a301c7afcf/validate.py#L21

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

File: config/env_sample.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+    "LOG_LEVEL": "WARN",
```
Comment (ssciolla) [inline config/env_sample.json]: This works, but I think the official level name is "WARNING".

File: validate.py [hunk 2]
```diff
@@ -18,7 +18,6 @@
 
 # Global variables
 
-logging.basicConfig(level="DEBUG")
```
Comment (ssciolla) [inline validate.py:21]: When I changed the `LOG_LEVEL` to "INFO", I wasn't seeing the `logger.info` calls from `execute_query_and_write_to_csv`. Changing them to `warning` made them show up. It seems to work as expected if you call this `basicConfig` method. So maybe we can keep this line but just use `logging.basicConfig()`? You probably know better how the `logging` module works than I do, but it was odd to me that I couldn't get the info messages to show up.

Comment (jonespm) [reply validate.py:21]: Ok adding in logging.baseConfig looks like it gets this working as expected. 
WARN works the same as WARNING. It looks like they're both supported but WARN was deprecated so updating that in the sample too.

**Review by ssciolla — APPROVED** [review]
Looks good!

**Pull request merged**